builder/main.go

```diff
@@ -23,7 +23,7 @@ func main() {
 
 	runner := buildpackrunner.New(&config)
 
-	err := runner.Run()
+	_, err := runner.Run()
```

Comment:
For visibility, the multi-buildpack uses that return value at https://github.com/cloudfoundry-incubator/multi-buildpack/blob/fa56d89bdc2a6502bd00126fc3db499e8b1119d3/src/compile/compile.go#L76, currently on the `use-multi-lifecycle` branch.